rhel7/profiles/anssi_nt28_high.profile

```diff
@@ -8,4 +8,12 @@ description: 'Draft profile for ANSSI compliance at the high level. ANSSI stands
 
 extends: anssi_nt28_enhanced
 
-selections: []
+selections:
+    - var_selinux_policy_name=targeted
+    - selinux_policytype
```

Comment:
NACK. The SELinux checks should be in the ``enhanced`` profile, not ``high``.

Reply:
SELinux in general is High level hardening for ANSSI.

Reply:
Can you point to documentation for that? According to ANSSI, SELinux is only for enhanced:

![image](https://user-images.githubusercontent.com/5713754/58036922-36b8bc80-7afa-11e9-8f10-7460888ec7e7.png)

Ref the French to English translation that was made available to Red Hatters:
https://docs.google.com/document/d/1zQiC2PLfpW4PV5zzIbzIwoYvKR-3dFFsw0yJ7AfS9Qg/edit

(apologies to the extended community since that isn't a public link)

Reply:
![image](https://user-images.githubusercontent.com/5713754/58037057-84cdc000-7afa-11e9-84a4-7131c9b2a117.png)

and

![image](https://user-images.githubusercontent.com/5713754/58037071-8dbe9180-7afa-11e9-8b75-ab3687609f37.png)


and

![image](https://user-images.githubusercontent.com/5713754/58037080-93b47280-7afa-11e9-9137-6f4700d77e87.png)

Reply:
![ANSSI_hardening_levels](https://user-images.githubusercontent.com/7460169/58075079-07ac5480-7ba7-11e9-9459-f0a885ffd430.png)
`E` doesn't stand for Enhanced. The letters probably wheren't translated.

Reply:
@shawndwells read through some of the document... @yuumasato is correct.